Makefile

```diff
@@ -18,28 +18,30 @@ all: format build test
 
 test:
 	@echo ">> running tests"
-	@$(GO) test -tags test -short -race $(pkgs)
+	$(GO) test -tags test -short -race $(pkgs)
 
-test-integration: build test
-	@./build/integration.sh
+test-integration:
+	./build/integration.sh
 
 format:
 	@echo ">> formatting code"
-	@$(GO) fmt $(pkgs)
+	$(GO) fmt $(pkgs)
 
 vet:
 	@echo ">> vetting code"
-	@$(GO) vet $(pkgs)
+	$(GO) vet $(pkgs)
 
 build:
 	@echo ">> building binaries"
-	@./build/assets.sh
-	@./build/build.sh
+	./build/assets.sh
```

Comment:
Yeah, do you think that's undesireable? I thought printing the commands that were executing was better for understanding what was happening, but I don't feel too strongly about it.